src/manager.js

```diff
@@ -414,71 +468,79 @@ AdManager.prototype.loadAds = function(element, updateCorrelator) {
 
     var slot = this.configureAd(thisEl);
 
-    if (typeof window.index_headertag_lightspeed === 'undefined') {
+    if (slot) {
+      slotsToLoad.push(slot);
+    }
+
+    if (typeof window.headertag === 'undefined' || window.headertag.apiReady !== true) {
       this.googletag.display(thisEl.id);
     } else {
-      window.index_headertag_lightspeed.slotDisplay(thisEl.id, ads);
+      window.headertag.display(thisEl.id);
```

Comment:
Note: Seems this option is not needed for index under headertag as opposed to lightspeed.